packages/core/src/components/ColorPalette/ColorPicker.tsx

```diff
@@ -29,11 +29,13 @@ function DropdownInput({
   colorPickerColor,
   format,
   onSetFormat,
+  shadow,
```

Comment:
```suggestion
  shadow = false,
```

Reply:
this way we don't have to prop drill `shadows` everywhere. as 99% of cases don't need it